pyface/ui/wx/action/action_item.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, parent, menu, item, controller):
         action  = item.action
         label   = action.name
         kind    = _STYLE_TO_KIND_MAP[action.style]
-        longtip = action.description
+        longtip = action.description or action.tooltip
```

Comment:
This is the comment I was basing the patch on:

https://github.com/enthought/pyface/blob/master/pyface/action/action.py#L45
```
    #: A longer description of the action (used for context sensitive help etc).
    #: If no description is specified, the tooltip is used instead (and if there
    #: is no tooltip, then well, maybe you just hate your users ;^).
    description = Unicode
```